### PR TITLE
pypi files configured update #1

### PR DESCRIPTION
**Pull request by brentonford**
pypi configuration completed up to Upload:
http://peterdowns.com/posts/first-time-with-pypi.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
-Required:
+# Requirements
 * Python 3.5
 
-The Directory object can be 
+# The Directory object can be 
 * Initialised on an existing directory
 * Scan that directory
-* Be interigated for the file list
```
Comment (jarrodchesney) [inline README.md:10]: Interrogated

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    name='pydirectory',
+    packages=['pydirectory'],  # this must be the same as the name above
+    version='0.1',
+    description='A class representing a file system directory, that deletes on '
+                'garbage collect.',
+    author='Synerty',
+    author_email='jarrod.chesney@synerty.com',
+    url='https://github.com/Synerty/pydirectory',
+    # use the URL to the github repo
+    download_url='https://github.com/Synerty/pydirectory/tarball/0.1',
```
Comment (jarrodchesney) [inline setup.py:13]: The starting version should be 0.1.0

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from distutils.core import setup
+
+setup(
+    name='pydirectory',
+    packages=['pydirectory'],  # this must be the same as the name above
+    version='0.1',
```
Comment (jarrodchesney) [inline setup.py:6]: 0.1.0

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    description='A class representing a file system directory, that deletes on '
+                'garbage collect.',
+    author='Synerty',
+    author_email='jarrod.chesney@synerty.com',
+    url='https://github.com/Synerty/pydirectory',
+    # use the URL to the github repo
+    download_url='https://github.com/Synerty/pydirectory/tarball/0.1',
+    # I'll explain this in a second
+    keywords=['directory'],  # arbitrary keywords
```
Comment (jarrodchesney) [inline setup.py:15]: lame

**Review by jarrodchesney — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (brentonford) [reply setup.py:13]: version corrected.

File: setup.py (hunk 3 above)
Comment (brentonford) [reply setup.py:6]: version corrected.

File: setup.py (hunk 4 above)
Comment (brentonford) [reply setup.py:15]: it stated 'arbitrary'...

File: README.md (hunk 1 above)
Comment (brentonford) [reply README.md:10]: I thought that was what you meant

**Pull request merged**